contracts/libraries/Offer.sol

```diff
@@ -14,6 +14,7 @@ struct Offer {
     address makerToken;
     uint256 makerTokenAmount;
     uint256 minMakerTokenAmount;
+    bool allowContractCall;
```

Comment:
Maybe something like `allowContractSender` to avoid confusing it with "contract call"?

Reply:
I'm ok with the naming.

Reply:
I'm ok with both.

Reply:
Renamed fd1d45d